Sources/ClientRuntime/Telemetry/Metrics/Histogram.swift

```diff
@@ -0,0 +1,18 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/// A Histogram measures a value where the statistics are likely meaningful.
```

Comment:
nit: What does "likely meaningful" mean here? Maybe re-wording it might be good for clarification?

Reply:
This is wording taken directly from the [OpenTelemetry Metrics API documentation](https://opentelemetry.io/docs/specs/otel/metrics/api/#histogram).